lib/ical/stringify.js

```diff
@@ -108,7 +108,15 @@ ICAL.stringify = (function() {
     var jsName = property[0];
     var params = property[1];
 
-    var line = name;
+    var groupName = params['group'];
```

Comment:
This stringifies both jCard and jCal. The group param does not work that way in jCal though. We may need to have a hook from the designSet that allows augmenting the property name.